_posts/getting-started/2000-01-01-migrate-from-shelly-cloud.md

```diff
@@ -25,6 +22,10 @@ On Scalingo, you don't need to specify the Ruby version "externally" to your pro
 
 ## domains
 
+To use your own domain on Shelly Cloud, you were used to add your domain to your `Cloudfile`.
+On Scalingo, you will have to add it either:<br>
```

Comment:
I meant https://translate.google.fr/?hl=fr#en/fr/Either%20by%20doing%20this%20or%20this
However I'm not sure how to use it..